### ISSUE TITLE
api url configuration is not consistently used

### ISSUE THREAD

**Issue by lukavdplas** [UUDigitalHumanitieslab/I-analyzer#1048]
Some frontend modules use an environment variable `apiUrl` to get API urls. (default value is `'/api/'`). However, corpus images and the elasticsearch service don't depend on this at all, so if you actually wanted to redirect to a different route, some crucial requests would fail.

This does make me wonder if there is any reason why you would want to configure the value of `apiUrl` in the first place. If you wanted to direct somewhere else, the proxy configuration would be the place to look, I think. Is there a use case for this?

**Comment by BeritJanssen** [comment]
Good point! I think this is a remainder of when all backend calls would be handled through the `/api/` route. None of the deployment templates seem to do anything with the api route either.
